packages/nextjs/app/builders/0x5D09525B883020C65A2B5cd017FFbE51B6B6c58F/page.tsx

```diff
@@ -0,0 +1,82 @@
+import type React from "react";
+import { Inter } from "next/font/google";
+import Image from "next/image";
+import Link from "next/link";
+import type { NextPage } from "next";
+import { FaEnvelopeSquare, FaGithub, FaTelegram } from "react-icons/fa";
+
+const inter = Inter({
+  subsets: ["latin"],
+  display: "swap",
+  variable: "--font-inter",
+  fallback: ["ui-sans-serif"],
+});
+
+const Profile: NextPage = () => {
+  return (
+    <div className={`${inter.variable} font-serif col-span-1 grid grid-cols-1 m-auto justify-center`}>
+      <div className="grid grid-cols-1 px-6 w-full max-w-4xl my-0 ">
+        <div className="col-span-1 flex-col max-w-4xl mx-auto overflow-hidden">
+          <div className="bg-base-100 border-base-300 border shadow-md shadow-secondary rounded-3xl px-6 lg:px-8 my-6 space-y-1 py-4 sm:px-6 bg-gradient-to-r dark:from-sky-950 dark:to-slate-500 from-sky-800 to-base-300 flex">
+            <div className="flex-col flex-1 mt-2">
+              <h1 className="mt-2 text-3xl font-bold text-white">XanDev</h1>
+              <p className="mt-1 max-w-2xl text-sm text-white">Software Developer & Web3 Engineer</p>
+            </div>
+            <Image
+              className="rounded-lg justify-end flex-0"
+              src="/buidlguidl_punk.png"
+              width={100}
+              height={100}
+              alt="Avatar"
+            />
```

Comment:
I got this warning in the Chrome DevTools console:
> hook.js:608 Image with src "/buidlguidl_punk.png" has either width or height modified, but not the other. If you use CSS to change the size of your image, also include the styles 'width: "auto"' or 'height: "auto"' to maintain the aspect ratio.

Make sure to always check your console for errors and warnings.

In this case, it looks you are squeezing the image into a wrong aspect-ratio.